src/HttpService.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
```

Comment:
I would leave this as a note somewhere for a TODO, if we ever do typescript or something... but each `Service` has a `create(client: Room.Client): Service` factory method.

Reply:
added to comments for now